test/multinode_test.go

```diff
@@ -360,24 +360,8 @@ func TestCloseDisconnectedParticipantOnSignalClose(t *testing.T) {
 	c1 := createRTCClient("c1", secondServerPort, nil)
 	waitUntilConnected(t, c1)
 
-	c2 := createRTCClient("c2", defaultServerPort, &client.Options{
-		SignalRequestInterceptor: func(msg *livekit.SignalRequest, next client.SignalRequestHandler) error {
-			switch msg.Message.(type) {
-			case *livekit.SignalRequest_Offer, *livekit.SignalRequest_Answer, *livekit.SignalRequest_Leave:
-				return nil
-			default:
-				return next(msg)
-			}
-		},
-		SignalResponseInterceptor: func(msg *livekit.SignalResponse, next client.SignalResponseHandler) error {
-			switch msg.Message.(type) {
-			case *livekit.SignalResponse_Offer, *livekit.SignalResponse_Answer:
-				return nil
-			default:
-				return next(msg)
-			}
-		},
-	})
+	c2 := createRTCClient("c2", defaultServerPort, nil)
```

Comment:
Please have a look at this test and let me know if this needs to reverted. The previous version was not doing actual connection and dropping signal offer/answers on the floor.

There was still a participant update because of not needing to be ACTIVE. But, that does not work now. So, have to do the `waitTillConnected`.

With this change, the only use of `Signal(Request|Response)Interceptor` is not using it anymore. But, not removing the interceptor code though.